### ISSUE TITLE
Better result for VQA

### ISSUE THREAD

**Issue by yixuan-qiao** [OFA-Sys/OFA#34]
Would you mind share some tricks to get ~0.4 improvements (80.04 vs 80.45) using current codebase on VQA? Probably not due to Shuffle the Training Data.

**Comment by yangapku** [comment]
Hi Yixuan, if the GPU memory enables, you can try to increase the image resolution (modify `--patch-image-size` to a larger value). However, the amount of computation and training step-time will also be significantly increased. The benefit of shuffling is already included in obtaining the result of 80.04. More improvement may be related to further pretraining, which you can try after the code of OFA pretraining is released (planned in this month). If you discover more new strategies during using OFA, we also strongly welcome you to share them with us :)
